`vpm compat .` at airs-bp3 baseline schema panics

`vpm compat .` at `airsbp3\baseline_schemas\pkg\github.com\untillpro\airs-bp3\apps\untill\airsbp` ->panic:
```
panic: interface conversion: interface is nil, not appdef.IType

goroutine 20 [running]:
github.com/voedger/voedger/pkg/appdefcompat.buildTreeNode(0x0?, {0x0?, 0x0})
        C:/workspace/go/gopath/pkg/mod/github.com/voedger/voedger@v0.0.0-20231228153302-0c400427e6b9/pkg/appdefcompat/impl.go:69 +0x186
github.com/voedger/voedger/pkg/appdefcompat.buildTree(...)
        C:/workspace/go/gopath/pkg/mod/github.com/voedger/voedger@v0.0.0-20231228153302-0c400427e6b9/pkg/appdefcompat/impl.go:48
github.com/voedger/voedger/pkg/appdefcompat.checkBackwardCompatibility({0x0?, 0x0?}, {0xfbaca0?, 0xc000664420})
        C:/workspace/go/gopath/pkg/mod/github.com/voedger/voedger@v0.0.0-20231228153302-0c400427e6b9/pkg/appdefcompat/impl.go:26 +0x45
github.com/voedger/voedger/pkg/appdefcompat.CheckBackwardCompatibility(...)
        C:/workspace/go/gopath/pkg/mod/github.com/voedger/voedger@v0.0.0-20231228153302-0c400427e6b9/pkg/appdefcompat/provide.go:12
main.compat(0x1?, {{0xc0000fa5b0, 0x66}, {0x11ef4b0, 0x1}, {0x0, 0x0}}, {0x0, 0x0, 0x0})
        C:/workspace/go/gopath/pkg/mod/github.com/voedger/voedger@v0.0.0-20231228153302-0c400427e6b9/cmd/vpm/compat.go:65 +0x28f
main.newCompatCmd.func1(0xc00014e400?, {0xc000090640?, 0x4?, 0xf1387c?})
        C:/workspace/go/gopath/pkg/mod/github.com/voedger/voedger@v0.0.0-20231228153302-0c400427e6b9/cmd/vpm/compat.go:43 +0x1b2
github.com/spf13/cobra.(*Command).execute(0xc00015a600, {0xc000090610, 0x1, 0x1})
        C:/workspace/go/gopath/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:940 +0x87c
github.com/spf13/cobra.(*Command).ExecuteC(0xc00015a900)
        C:/workspace/go/gopath/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3a5
github.com/spf13/cobra.(*Command).Execute(...)
        C:/workspace/go/gopath/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:992
github.com/spf13/cobra.(*Command).ExecuteContext(...)
        C:/workspace/go/gopath/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:985
github.com/untillpro/goutils/cobrau.ExecCommandAndCatchInterrupt.func1({0xfb63a8?, 0xc0000e8550?})
        C:/workspace/go/gopath/pkg/mod/github.com/untillpro/goutils@v0.0.0-20231117130619-11fe1908c171/cobrau/catch_interrupt.go:21 +0x50
github.com/untillpro/goutils/cobrau.goAndCatchInterrupt.func1()
        C:/workspace/go/gopath/pkg/mod/github.com/untillpro/goutils@v0.0.0-20231117130619-11fe1908c171/cobrau/catch_interrupt.go:40 +0x6e
created by github.com/untillpro/goutils/cobrau.goAndCatchInterrupt in goroutine 1
        C:/workspace/go/gopath/pkg/mod/github.com/untillpro/goutils@v0.0.0-20231117130619-11fe1908c171/cobrau/catch_interrupt.go:38 +0x171
```

--- a/cmd/vpm/compat.go
+++ b/cmd/vpm/compat.go
@@ -62,17 +62,16 @@ func compat(compileRes *compileResult, params vpmParams, ignores [][]string) err
 		errs = append(errs, coreutils.SplitErrors(err)...)
 	}
 
-	compatErrs := appdefcompat.CheckBackwardCompatibility(baselineAppDef, compiledAppDef)
-	compatErrs = appdefcompat.IgnoreCompatibilityErrors(compatErrs, ignores)
-	if len(compatErrs.Errors) > 0 {
+	if baselineAppDef != nil && compiledAppDef != nil {
+		compatErrs := appdefcompat.CheckBackwardCompatibility(baselineAppDef, compiledAppDef)
+		compatErrs = appdefcompat.IgnoreCompatibilityErrors(compatErrs, ignores)
 		errObjs := make([]error, len(compatErrs.Errors))
 		for i, err := range compatErrs.Errors {
 			errObjs[i] = err
 		}
 		errs = append(errs, errObjs...)
-		return errors.Join(errs...)
 	}
-	return nil
+	return errors.Join(errs...)
 }
 
 // readIgnoreFile reads yaml file and returns list of errors to be ignored

--- a/pkg/utils/utils.go
+++ b/pkg/utils/utils.go
@@ -54,8 +54,11 @@ func PartitionID(wsid istructs.WSID, numCommandProcessors CommandProcessorsCount
 }
 
 func SplitErrors(joinedError error) (errs []error) {
-	if e, ok := joinedError.(interface{ Unwrap() []error }); ok {
-		return e.Unwrap()
+	if joinedError != nil {
+		if e, ok := joinedError.(interface{ Unwrap() []error }); ok {
+			return e.Unwrap()
+		}
+		return []error{joinedError}
 	}
 	return
 }

